delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/marshalling/RdfExceptionHandler.scala

```diff
@@ -32,11 +33,12 @@ object RdfExceptionHandler {
       base: BaseUri
   ): ExceptionHandler =
     ExceptionHandler {
-      case err: IdentityError  => discardEntityAndForceEmit(err)
-      case err: AuthTokenError => discardEntityAndForceEmit(err)
-      case AuthorizationFailed => discardEntityAndForceEmit(AuthorizationFailed: ServiceError)
-      case err: RdfError       => discardEntityAndForceEmit(err)
-      case err: Throwable      =>
+      case err: IdentityError        => discardEntityAndForceEmit(err)
+      case err: PermissionsRejection => discardEntityAndForceEmit(err)
```

Comment:
Neither solution is great but so far we tend to do a `attemptNarrow` to have an `IO[Either[_, _]]` to handle with errors (see the routes of the already migrated parts)

A better solution long term (so not in this PR and probably in a while) would be to have the different modules declare their exception handler and to use.

Reply:
Yeah I was thinking about this. It's one of those situations where typeclasses don't work so well. I wonder if what we need is something more like a trait on `Rejection` where we "force" user-facing errors to provide the necessary typeclasses. That way it's forced by the compiler and is a bit more streamlined

Reply:
Sorry, I misread this. I'll merge this but I'll do the fix you mentioned in the next PR I'm working on now